docs/tactics.md

```diff
@@ -1138,10 +1138,29 @@ Here too, the `reassoc` attribute can be used instead. It works well when combin
 attribute [simp, reassoc] some_class.bar
 ```
 ### sanity_check
+User commands to spot common mistakes in the code
 
-The `#sanity_check` command checks for common mistakes in the current file or in all of mathlib, respectively.
+* `#sanity_check`: check all declarations in the current file
+* `#sanity_check_mathlib`: check all declarations in mathlib (excluding the current file)
```

Comment:
```suggestion
* `#sanity_check_mathlib`: check all declarations in mathlib (but not core, or the current project if run outside mathlib; also for technical reasons, skips the current file)
```

Reply:
I thought that was covered under the term `mathlib`, but I added a clarification.